src/pages/Main/Release/Release.js

```diff
@@ -0,0 +1,74 @@
+import Product from '../../../components/Product/Product';
+
+import './release.scss';
+
+function Release() {
+  return (
+    <div className="release">
+      <div className="header">
+        <h1 className="title">Release</h1>
+        <button className="linkButton">
+          더 많은 제품 보기&nbsp;&nbsp;&nbsp;>
+        </button>
+      </div>
+      <div className="carouselContainer">
+        {products.map(product => (
+          <Product
+            key={product.id}
+            tag={product.tag}
+            desc={product.desc}
+            price={product.price}
+            imgUrl={product.imgUrl}
+            colorList={product.colorList}
+            sizeList={product.sizeList}
+          />
+        ))}

```

Comment:
```suggestion
{products.map(product => {
          const { id, tag, desc, price, imgUrl, colorList, sizeList } = product;
          return (
            <Product
              key={id}
              tag={tag}
              desc={desc}
              price={price}
              imgUrl={imgUrl}
              colorList={colorList}
              sizeList={sizeList}
            />
          );
        })}
```
return 문을 작성하면 그 위에서 구조분해 할당 할 수 있습니다!